src/woo_publications/publications/api/serializers.py

```diff
@@ -171,6 +180,39 @@ def validate_publicatiestatus(
 
         return value
 
+    def get_or_create_organisation_member(self, owner: EigenaarSerializer | None):
+        identifier = self.context["request"].headers[AUDIT_USER_ID_PARAMETER.name]
+        naam = self.context["request"].headers[AUDIT_USER_REPRESENTATION_PARAMETER.name]
+
+        if owner:
+            identifier = owner["identifier"]
+            naam = owner["naam"]
+
+        org_member, _ = OrganisationMember.objects.get_or_create(
+            identifier=identifier,
+            naam=naam,
+        )
+
+        return org_member
+
+    @transaction.atomic
+    def update(self, instance, validated_data):
+        if "eigenaar" in validated_data:
+            eigenaar = validated_data.pop("eigenaar")
+            org_member, _ = OrganisationMember.objects.get_or_create(
+                identifier=eigenaar["identifier"],
+                naam=eigenaar["naam"],
+            )
```

Comment:
I think you want to use `update_or_create`, and do the lookup on `identifier` only, not both `identifier` and `naam`.